components/project-card/project-card.scss

```diff
@@ -51,6 +51,11 @@
     margin: 0.5rem 0;
   }
 
+  p {
+    font-size: 0.9375rem;
```

Comment:
Yea I wasn't sure if it was intended. The paragraph text in (detail view) project card is 15px and with line height set to 1.333. Whereas default body text is 16px + line height 1.5.